Cargo.toml

```diff
@@ -65,14 +65,14 @@ windows-targets = "0.52"
 # wasm_js
 [target.'cfg(all(getrandom_backend = "wasm_js", target_arch = "wasm32", any(target_os = "unknown", target_os = "none")))'.dependencies]
 wasm-bindgen = { version = "0.2.96", default-features = false }
-js-sys = "0.3.73"
+js-sys = { version = "0.3.73", default-features = false }
```

Comment:
I think we only need this line and all other changes can be removed.

Reply:
I do really appreciate that you are pushing back on this actually.
Thanks!